codegen/smithy-kotlin-codegen/src/test/kotlin/software/amazon/smithy/kotlin/codegen/core/SymbolProviderTest.kt

```diff
@@ -529,21 +529,58 @@ class SymbolProviderTest {
 
         assertEquals("Map<String, Record>", mapSymbol.name)
 
-        // collections should contain a reference to the member type
-        assertEquals("Record", mapSymbol.references[0].symbol.name)
+        // collections should contain a reference to the member types
+        val refNames = mapSymbol.references.map { it.symbol.fullName }
+        assertTrue("kotlin.String" in refNames)
+        assertTrue("com.test.model.Record" in refNames)
 
         val sparseMapSymbol = provider.toSymbol(model.expectShape<MapShape>("${TestModelDefault.NAMESPACE}#MySparseMap"))
 
         assertEquals("Map<String, Record?>", sparseMapSymbol.name)
 
         // collections should contain a reference to the member type
-        assertEquals("Record", sparseMapSymbol.references[0].symbol.name)
+        val sparseRefNames = mapSymbol.references.map { it.symbol.fullName }
```

Comment:
question: is this supposed to use `sparseMapSymbol` instead of `mapSymbol`?

Reply:
Yes it is, good catch!